### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
Closes: #82

Hi [tungbq](https://github.com/tungbq),
just trying to add `getting-started` link on [the content](https://github.com/tungbq/devops-basic#content:),
please check it out

## What is the purpose of the change

add getting-started to the content table in README.md
[Update README.md](https://github.com/tungbq/devops-basic/commit/a246f9a0e4f9e5629936d3729404631fed6aa765)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,30 +32,30 @@
...
-      <td align="center" valign="top" width="14.28%"><a href="https://xzm27-d5f48.web.app"><img src="https://avatars.githubusercontent.com/u/55028818?v=4?s=100" width="100px;" alt="XZM27"/><br /><sub><b>XZM27</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=Xzm27" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/SaketKaswa20"><img src="https://avatars.githubusercontent.com/u/105808363?v=4?s=100" width="100px;" alt="Saket Kaswa"/><br /><sub><b>Saket Kaswa</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=SaketKaswa20" title="Code">💻</a></td>
-    </tr>
-  </tbody>
-</table>
-
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
-
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
```
Comment (tungbq) [inline README.md:35]: @viktoriussuwandi it seems the content of contributors section were changed too. Could you please check and only change the `gertting-started` part and keep other part as is?

Comment (viktoriussuwandi) [reply README.md]: yes sure

File: README.md [hunk 2]
```diff
@@ -65,3 +66,8 @@
 <!-- prettier-ignore-end -->
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```
Comment (tungbq) [inline README.md:73]: Please do not change these lines - @viktoriussuwandi . It seems duplicated, could you please check?

Comment (viktoriussuwandi) [reply README.md]: sorry @tungbq, the duplicate lines removed

Comment (tungbq) [reply README.md]: Thanks @viktoriussuwandi

**Pull request merged**